web-service/static/style.css

```diff
@@ -0,0 +1,134 @@
+html {
```

Comment:
This is a sort of common thing - treating the UI related changes as not worth breaking apart. I suggest talking in the commit message a bit about why you choose some of the styling, even if it works out to something like "stole bootstrap.css".